src/editors/components/EditorHeader/HeaderTitle.jsx

```diff
@@ -16,7 +16,7 @@ export const HeaderTitle = ({
   intl,
   isInitialized,
   setBlockTitle,
-  typeHeader,
+  returnTitle,
```

Comment:
I'm a little confused what this variable does based on what I thought it was versus its name. Is this the place where a "new" title is set to?

Reply:
This was a selector that pulled the block type and the reason why we were seeing 'Text' as the title. I checked where this was used and it was only used to set up the title. Because of that, I choose to update the logic and change its name to something more apt.

Reply:
Ok makes sense! TY